hw/ip/otbn/data/insns.yml

```diff
@@ -491,6 +473,72 @@ insns:
       elif writeback_variant == 'writeout':
         WDR[d] = ACC
 
+  - mnemonic: bn.mulqacc.wo
+    group: bignum
+    synopsis: Quarter-word Multiply and Accumulate with half-word writeback
+    operands:
+      - *mulqacc-zero-acc
+      - &mulqacc-wrd
+        name: wrd
+        doc: Destination WDR.
+      - *mulqacc-wrs1
+      - *mulqacc-wrs1-qwsel
+      - *mulqacc-wrs2
+      - *mulqacc-wrs2-qwsel
+      - *mulqacc-acc-shift-imm
+    syntax: |
+      [<zero_acc>] <wrd>, <wrs1>.<wrs1_qwsel>, <wrs2>.<wrs2_qwsel>, <acc_shift_imm>
+    doc: |
+      Multiplies two `WLEN/4` WDR values and adds the result to an accumulator after shifting it.
+      Writes the resulting accumulator to <wrd>.
+    decode: |
+      writeback_variant = 'writeout'
+      zero_accumulator = DecodeMulqaccZeroacc(zero_acc)
+
+      d = UInt(wrd)
+      a = UInt(wrs1)
+      b = UInt(wrs2)
+
+      d_hwsel = None
+      a_qwsel = DecodeQuarterWordSelect(wrs1_qwsel)
+      b_qwsel = DecodeQuarterWordSelect(wrs2_qwsel)
+    operation: *mulqacc-operation
+
+  - mnemonic: bn.mulqacc.so
+    group: bignum
+    synopsis: Quarter-word Multiply and Accumulate with half-word writeback
```

Comment:
Can you rephrase that to be different from the .wo variant, and include "shift" in it somewhere?